package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "material-ui": "next",
     "material-ui-icons": "^1.0.0-beta.36",
+    "nodemon": "^1.17.2",
```

Comment:
Doesn't need to be in dependencies for react app.

Reply:
Will fix that.